### PR TITLE
disable custom ciphers by default

### PR DESCRIPTION
**Pull request by cthtrifork**

### CODE REVIEW

**Review by pegtrifork — APPROVED** [review]
Approved when linting is fixed

File: charts/flink-job/values.yaml [hunk 1]
```diff
@@ -33,11 +33,14 @@ version: v1_16
...
   # -- Whether to use SSL between the job- and taskmanager
   enabled: true
-  # -- Set the algorithms allowed.
-  # see also: <https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/security/security-ssl/#cipher-suites>
-  algorithms: "TLS_CHACHA20_POLY1305_SHA256,TLS_AES_256_GCM_SHA384"
-  # -- Set the protocol allowed.
-  protocol: "TLSv1.3"
+  customCiphers:
+    # -- Whether to enable custom ciphers
+    enabled: false
```
Comment (cthtrifork) [inline charts/flink-job/values.yaml:38]: Should it be default true or false 🤔

**Comment by cthtrifork** [comment]
> Approved when linting is fixed

Fixed 😓

File: charts/flink-job/values.yaml (hunk 1 above)
Comment (pegtrifork) [reply charts/flink-job/values.yaml:38]: False. By default it should be controlled by the operator

**Pull request merged**